.pre-commit-config.yaml

```diff
@@ -64,3 +64,10 @@ repos:
         exclude: >-
           ^docs/changelog/(\d+\.(bugfix|feature|deprecation|breaking|doc|misc).rst|README.rst|template.jinja2)
         files: ^docs/changelog/
+      - id: changelogs-user-role
+        name: changelog files have a non-broken :user:`name` role
+        language: pygrep
+        entry: >-
+          :user:([^`]+`?|`[^`]+[\s,])
```

Comment:
there's no reason to bring `>-` into this imo -- just do it inline

also this is probably just as good as the more complicate regex:

```
entry: ':user:[^`]'
```

Reply:
@asottile my regex
```
docs/changelog.rst:25:  provision metadata are written as JSON to that path - by :user:
docs/changelog.rst:26:  provision metadata are written as JSON to that path - by :user:hroncok`
docs/changelog.rst:27:  provision metadata are written as JSON to that path - by :user:hroncok
docs/changelog.rst:28:  provision metadata are written as JSON to that path - by :user:`hroncok
```
your regex
```
docs/changelog.rst:25:  provision metadata are written as JSON to that path - by :user:
docs/changelog.rst:26:  provision metadata are written as JSON to that path - by :user:hroncok`
docs/changelog.rst:27:  provision metadata are written as JSON to that path - by :user:hroncok
```

Reply:
> there's no reason to bring `>-` into this imo -- just do it inline

we clearly prefer different styles :man_shrugging:

Reply:
I'd also say make it inline but use the more advanced regex 🙏

Reply:
Done.